web/firefox_print_service.js

```diff
@@ -33,14 +32,8 @@ function composePage(
   canvas.width = Math.floor(size.width * PRINT_UNITS);
   canvas.height = Math.floor(size.height * PRINT_UNITS);
 
-  // The physical size of the canvas as specified by the PDF document.
-  canvas.style.width = Math.floor(size.width * CSS_UNITS) + "px";
-  canvas.style.height = Math.floor(size.height * CSS_UNITS) + "px";
-
```

Comment:
Given that this is being removed, I cannot help but wonder if we still need to keep the following code around?
https://github.com/mozilla/pdf.js/blob/63471bcbbeee3fc1c6b1df988174640a124f097a/web/pdf_print_service.js#L39-L43

---

Obviously I see that those values are being [returned below](https://github.com/mozilla/pdf.js/blob/63471bcbbeee3fc1c6b1df988174640a124f097a/web/pdf_print_service.js#L63-L68), but I don't think that we use them now that you're also removing https://github.com/mozilla/pdf.js/blob/63471bcbbeee3fc1c6b1df988174640a124f097a/web/pdf_print_service.js#L184-L185 in this patch.

Reply:
Yup, you're right this can be cleaned up, good catch!